### PR TITLE
Add test to load ashe sample

### PR DESCRIPTION
**Pull request by andrewmil**
Added test to load ashe sample. This is different from other loading samples as the sample file contains considerably more empty fields

Depends on other branches
[rm-sample-service](https://github.com/ONSdigital/rm-sample-service/pull/24)
[rm-samplesvc-api](https://github.com/ONSdigital/rm-samplesvc-api/pull/6)
[rm-survey-service](https://github.com/ONSdigital/rm-survey-service/pull/28)

[Trello](https://trello.com/c/Fq5Xv2oW/155-us081-int-load-ashe-sample-for-r18-m)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/259522840/US081+INT+-+Load+ASHE+Sample)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US081: INT - Load ASHE Sample for R18 (M)](https://trello.com/c/Fq5Xv2oW/155-us081-int-load-ashe-sample-for-r18-m)

File: acceptance_tests/features/load_ashe_sample_file.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+   As a Collection Exercise Coordinator
+   I need to be able to load an ASHE sample
+   So that a sample file exists in the system
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  @us081_s001
+  Scenario: Load ASHE sample
+    Given the 201803 collection exercise for the QIFDI survey has been created
```
Comment (tej99) [inline acceptance_tests/features/load_ashe_sample_file.feature]: the 201803 collection exercise for the **ASHE** survey has been created

File: acceptance_tests/features/pages/collection_exercise_details.py [hunk 2]
```diff
@@ -32,6 +32,11 @@ def load_sample():
     browser.find_by_id('btn-load-sample').click()
 
 
+def load_ashe_sample():
+    test_file = 'resources/sample_files/us081-load_ashe_sample_file/ashe_sample.csv'
```
Comment (tej99) [inline acceptance_tests/features/pages/collection_exercise_details.py]: Refactoring the existing functions may be better

```
def select_sample(sample_file_path):
     browser.driver.find_element_by_id('sampleFile').send_keys(abspath(sample_file_path))
 
 
 def load_sample(sample_file_path):
     select_sample(sample_file_path)
     browser.find_by_id('btn-load-sample').click()
```

File: resources/database/collection_exercise_setup/ashe_201803_setup.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+values('46a4cdc2-8995-4969-a02d-37f6be767efa',8,'ASHE','Annual Survey of Hours and Earnings','SYSTEM',now());
+
+insert into collectionexercise.casetypedefault(casetypedefaultpk, sampleunittypefk, actionplanid, survey_uuid)
+values(7,'B','878c3462-766a-4cf6-8fb3-867d5124cc0a','6aa8896f-ced5-4694-800c-6cd661b0c8b2');
+
+insert into collectionexercise.casetypedefault(casetypedefaultpk, sampleunittypefk, actionplanid, survey_uuid)
+values(8,'BI','46a4cdc2-8995-4969-a02d-37f6be767ef4','6aa8896f-ced5-4694-800c-6cd661b0c8b2');
+
+update collectionexercise.collectionexercise
+ set scheduledstartdatetime = '2017-09-11 23:00:00+00',
```
Comment (tej99) [inline resources/database/collection_exercise_setup/ashe_201803_setup.sql]: Are the collection exercise events for Ashe not being loaded by the rm-tools collex loader?

Comment (tej99) [reply resources/database/collection_exercise_setup/ashe_201803_setup.sql]: Looks like it is being loaded in, I don't think this is needed.

**Pull request merged**